tests/models/conftest.py

```diff
@@ -13,7 +13,38 @@
 
 import pytest
 
+from andromede.model.parsing import parse_yaml_library
+from andromede.model.resolve_library import resolve_library
+
 
 @pytest.fixture(scope="session")
 def libs_dir() -> Path:
     return Path(__file__).parent / "libs"
+
+
+@pytest.fixture(scope="session")
+def data_dir() -> Path:
+    return Path(__file__).parents[1] / "unittests/data"
+
+
+@pytest.fixture(scope="session")
+def lib(data_dir: Path):
+    lib_file = data_dir / "lib.yml"
+
+    with lib_file.open() as f:
+        input_lib = parse_yaml_library(f)
+
+    lib = resolve_library(input_lib)
+    return lib
+
+
+@pytest.fixture(scope="session")
+def lib_sc(data_dir: Path):
```

Comment:
Replace `data_dir` argument which is unused by `libs_dir`, so that `libs_sc_file = libs_dir / "standard_sc.yml"`